indico/modules/events/persons/templates/management/_person_list_base.html

```diff
@@ -72,6 +72,37 @@
                         </ul>
                     </div>
                 {% endif %}
+                <div class="group">
+                    <a id="no-account-dropdown" href="#" class="i-button arrow highlight disabled js-dropdown" data-toggle="dropdown">
+                        {% trans %}Users with no account{% endtrans %}
+                    </a>
+                    <ul class="dropdown">
+                        <li>
+                            <a href="#"
+                               title="{% trans %}Send an email to all users with no Indico account{% endtrans %}"
+                               data-href="{{ url_for('persons.email_event_persons', event, no_account = True) }}"
```

Comment:
`no_account=true` - bools/none are lowercase in jinja and kwargs have no space around the `=`